### PR TITLE
fix: unskip tests in apply_test.go and get them passing

### LINKED ISSUE #7537: re-enable integration/apply_test.go
##Description

integration some tests in integration/apply_test.go were temporarily disabled due to v2 migration without completing v2 render
Some disabled tests may fail when enabling it due to test skaffold.yaml don't have render config setup in skaffold.yaml, adding that part should fix the issue.
now v2 render is complete, as the goal of v2.0 alpha 1 milestone, we should make all integration tests pass
##ASK

re-enable all tests in integration/apply_test.go

### PR DESCRIPTION
**Pull request by aaron-prindle**
Fixes #7537

### CODE REVIEW

**Comment by aaron-prindle** [comment]
Currently failing with:
```
=== Failed Tests ===
/integration/TestRenderApplyHelmDeployment
--- FAIL: TestRenderApplyHelmDeployment (16.07s)
/integration/TestRenderApplyHelmDeployment/DiagnoseRenderApply
    util.go:95: Test partition: 3
time="2022-06-13T21:27:47Z" level=info msg="Namespace: skaffoldb8wmm" subtask=-1 task=DevLoop
    apply_test.go:63: Running [skaffold diagnose --yaml-only] in examples/helm-deployment
    apply_test.go:63: Ran [skaffold diagnose --yaml-only] in 158.767643ms
    apply_test.go:70: Running [skaffold render --namespace skaffoldb8wmm --default-repo gcr.io/k8s-skaffold --digest-source=local -f skaffold-diagnose.yaml] in
    apply_test.go:70: Ran [skaffold render --namespace skaffoldb8wmm --default-repo gcr.io/k8s-skaffold --digest-source=local -f skaffold-diagnose.yaml] in 15.12 seconds
    helper.go:259: Running [skaffold apply --namespace skaffoldb8wmm render.yaml -f skaffold-diagnose.yaml] in
    helper.go:259: skaffold apply: exit status 1,
    panic.go:642: render.yaml is not a valid Kubernetes manifest
    --- FAIL: TestRenderApplyHelmDeployment/DiagnoseRenderApply (16.07s)
```

File: integration/examples/helm-deployment/skaffold.yaml [hunk 1]
```diff
@@ -3,7 +3,7 @@ kind: Config
 build:
   artifacts:
   - image: skaffold-helm
-deploy:
+manifests:
```
Comment (ericzzzzzzz) [inline integration/examples/helm-deployment/skaffold.yaml:6]: I vaguely remember that @marlon-gamez told me that we want to keep helm in the old way and use helm as deployer? If we change this line, does this mean we use helm for render and use kubectl for deploy?

Comment (aaron-prindle) [reply integration/examples/helm-deployment/skaffold.yaml:6]: Discussed w/ @MarlonGamez offline, having the schema use `manifests` is required to have the helm renderer work as expected (which is required for this test).

Comment (ericzzzzzzz) [reply integration/examples/helm-deployment/skaffold.yaml:6]: ohhh I see. thanks.

**Review by ericzzzzzzz — APPROVED** [review]

**Pull request merged**